src/i18n_xml_from_excel.py

```diff
@@ -28,10 +30,8 @@ def getCleanString(input):
 	input = input.replace('...', '&#8230;')
 	input = input.replace(u'…', '&#8230;')
```

Comment:
This is my own legacy code but... 2 years ago I preferred html entity over unicode char. Now I prefer unicode char :smile: So I would remove this line and replace three dots with the ellipsize char alone, that I think is way more descriptive.

```python
input = input.replace('...', u'…')
```

Also, I'd extract all this `input.replace` stuff into a `sanitize` method, for example.